src/components/Login/ProfileSettings.tsx

```diff
@@ -153,8 +125,9 @@ export function ProfileSettings({ source, onClose }: ProfileSettingsProps) {
                 <ClickableButton
                     className="flex items-center overflow-hidden whitespace-nowrap rounded px-2 py-3 hover:bg-bg md:mb-3"
                     onClick={() => {
+                        const account = accounts.find((x) => isSameProfile(x.profile, currentProfile));
                         LogoutModalRef.open({
-                            account: accounts.find((x) => isSameProfile(x.profile, currentProfile)),
+                            account: account?.session ? (account as Account) : undefined,
```

Comment:
Make `useConnectedAccounts` returns `account` type.

```suggestion
                            account: account?.session ? (account) : undefined,
```